### ISSUE TITLE
Cannot read property 'length' of undefined, Ember 3.5

### ISSUE THREAD

**Issue by allthesignals** [kaliber5/ember-cli-bundle-analyzer#8]
I have jQuery disabled, seeing this: 

```
(node:94431) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'length' of undefined
    at compress.then.compressed (/Users/mgardner/labs-zola/node_modules/broccoli-concat-analyser/lib/summarize.js:54:29)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
(node:94431) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:94431) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by allthesignals** [comment]
Woops, seems like a duplicate of https://github.com/stefanpenner/broccoli-concat-analyser/issues/32. Will try the fixes listed there... not sure if this is something new or a dependency issue.

**Comment by simonihmig** [comment]
Pretty sure this is a problem in the upstream `broccoli-concat-analyser`, but will try to investigate this...

> It seems to die with it uglifies moment/lib

Hm, yes, could indeed be related to failing uglification...

@allthesignals Is there a way to reproduce this easily? Is your project in the open, or have some minimal reproduction? `moment/lib` comes from `ember-moment`?

**Comment by allthesignals** [comment]
Hi @simonihmig, thanks for the response. The repo is open, I'm seeing the behavior here: https://github.com/NYCPlanning/labs-zola

I also have trouble getting `broccoli-concat-analyser` to generate the dot folder:

```bash
$ broccoli-concat-analyser .
⠹ processing...TypeError: Cannot convert undefined or null to object
```

**Comment by simonihmig** [comment]
Ah, thanks Matt, that was really helpful. Also nice to see you are developing this app in the open! 😄 

I debugged this, and uncovered quite a few different issues: 😬
* #9 
* #10
* https://github.com/stefanpenner/broccoli-concat-analyser/issues/39
* https://github.com/stefanpenner/broccoli-concat-analyser/issues/40

The root issue is indeed that `ugify-es` was failing to minify your `vendor.js` with some error during AST-traversal. It did work with some older version of it though (which I still had in my `yarn.lock`). Proof:
![image](https://user-images.githubusercontent.com/1325249/47964212-9f48bc00-e036-11e8-8bf0-d138ffd8ed32.png)

Will walk through the issues asap...

**Comment by simonihmig** [comment]
I fixed most things, and published `v0.0.4`. With that version I could successfully get the stats output in your app! 🎉

I hope it works for you as well. Closing this for now, feel free to reopen if it doesn't!

**Comment by allthesignals** [comment]
Thank you so much @simonihmig! You are lightning fast! My treemap works great. Now back to tracking down oversized assets...

I think my only separate issue is [here](https://github.com/stefanpenner/broccoli-concat-analyser/issues/34), but I'm at least seeing the file sizes match up:

<img width="1552" alt="screen shot 2018-11-05 at 9 43 54 pm" src="https://user-images.githubusercontent.com/5004319/48039682-e96b9380-e143-11e8-9d86-f80743291c3e.png">
